docs/content/guides/dagster/branch_depl_with_snowflake.mdx

```diff
@@ -0,0 +1,325 @@
+---
+title: Branch Deployments with Snowflake | Dagster
+description: This guide illustrates a workflow that enables testing Dagster code in your cloud environment without impacting your production data.
+---
+
+# Branch Deployments with Snowflake
+
+This guide details a workflow to test Dagster code in your cloud environment without impacting your production data. To highlight this functionality, we’ll leverage Dagster Cloud branch deployments and a Snowflake database to:
+
+- Execute code on a feature branch directly on Dagster Cloud
+- Read and write to a clone of our Snowflake data
+
+With these tools, we can merge changes with confidence in the impact on our data platform and with assurance that our code will execute as intended.
+
+### Prerequisites
+
+<Note>
+  This guide is an extension of the{" "}
+  <a href="https://docs.dagster.io/guides/dagster/local-to-production-tutorial">
```

Comment:
```suggestion

  <Link href="/guides/dagster/local-to-production-tutorial">
```
we should use relative link and use `<Link>` like here: https://github.com/dagster-io/dagster/blame/master/docs/content/deployment/overview.mdx#L40-L42 - so the internal reference can be handled by versioning and won't break in the future